src/ResourceManagement/Batch/Batch.Tests/InMemoryTests/AccountTests.InMemory.cs

```diff
@@ -833,11 +833,10 @@ public void AccountKeysRegenerateThrowsExceptions()
             var handler = new RecordedDelegatingHandler();
             var client = BatchTestHelper.GetBatchManagementClient(handler);
 
-            Assert.Throws<ValidationException>(() => client.Account.RegenerateKey(null, "bar", new BatchAccountRegenerateKeyParameters()));
-            Assert.Throws<ValidationException>(() => client.Account.RegenerateKey("foo", null, new BatchAccountRegenerateKeyParameters()));
-            Assert.Throws<ValidationException>(() => client.Account.RegenerateKey("foo", "bar", null));
```

Comment:
@cormacpayne Because we turned flattening on, and the `KeyType` enumeration is a required parameter, so it's not possible for it to be null anymore because the signature is now `.Regeneratekey(string, string, AccountKeyType)` -- since the enum can't be null the test is now the compiler =)